actor/echo-messaging/actor.mk

```diff
@@ -0,0 +1,138 @@
+# common makefile rules for building actors
```

Comment:
We can remove the Makefile entirely now that wash build is up-to-snuff

Reply:
Actually, since it's referenced in the actions, I'm OK with fixing this in another PR for the rest of the repo